src/test/resources/checks/NoFunctionCallWhenDeclaringForLoop.php

```diff
@@ -26,11 +26,6 @@
     ++$i;
 }
 
-/* exemple 4 */
-
-for ($i = 1, $j = 0; $i <= 10; $j += $i, print $i, ++$i); // NOK {{Do not call a function in for-type loop declaration}}
```

Comment:
please keep this example, but add a bloc under for statement if you want. for me this use case is important to check because we want to verify if a call of a function is detected in the last part of for statement